plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -567,3 +612,18 @@ func (p *Plugin) admitCSINode(nodeName string, a admission.Attributes) error {
 
 	return nil
 }
+
+func (p *Plugin) allowedMirrorPodLabels(a admission.Attributes) (sets.String, error) {
+	ns, err := p.nsGetter.Get(a.GetNamespace())
+	if errors.IsNotFound(err) {
+		return nil, err
+	}
+	if err != nil {
+		return nil, admission.NewForbidden(a, err)
```

Comment:
add context to the error here and above ("error fetching namespace %q to determine allowed mirror pod labels: %v", etc)

Reply:
Removed.